packages/utils/src/loki.js

```diff
@@ -225,12 +241,13 @@ export class Logging {
   }
 
   /**
-   * Add log entry to batch
+   * Add log entry to batch after applying field filtering
    *
    * @param {any} body
    */
   _add(body) {
-    this.logEventsBatch.push(body)
+    const log = this.filterFields ? this.filterFields(body) : body
```

Comment:
I think this is the right idea, but I'm getting a bit hung up on the name `filterFields` - this callback could really be doing any sort of transformation here and it feels like the name should probably reflect that - maybe `logMessageTransformer` or something? It's a bit wordy, but it only gets written out when the `Logging` object gets set up so shouldn't have a huge practical impact on DX. Open to other names if you've got ideas!

Reply:
Thanks for the heads-up! I agree—it makes sense to clarify that this is a transformer. I suggest `logDataTransformer` to help differentiate it from the message field in the log object.